runtime/compilers/rillv1/parse_model.go

```diff
@@ -144,6 +147,31 @@ func (p *Parser) parseModel(node *Node) error {
 		}
 	}
 
+	// hack to mark model as updated when local file changes
+	if inputConnector == "local_file" {
+		path, ok := inputProps["path"].(string)
+		if ok {
+			root, err := p.Repo.Root(ctx)
+			if err != nil {
+				return err
+			}
+			localPaths, err := doublestar.FilepathGlob(filepath.Join(root, path))
+			if err == nil {
+				for _, localPath := range localPaths {
+					localPath = strings.TrimPrefix(localPath, root)
```

Comment:
Could this use `p.Repo.ListRecursive` instead of resorting to a file system dependency? Currently I don't believe the parser has any direct file system dependency (which may come in handy for cloud editing eventually)